pkg/gameserversets/gameserversets.go

```diff
@@ -52,8 +52,12 @@ func sortGameServersByLeastFullNodes(list []*agonesv1.GameServer, count map[stri
 				return false
 			}
 		}
-
-		return (ac.Allocated + ac.Ready) < (bc.Allocated + bc.Ready)
+		// if both Nodes have the same count, one node is emptied first (packed scheduling behavior)
+		acTotal, bcTotal := ac.Allocated+ac.Ready, bc.Allocated+bc.Ready
+		if acTotal == bcTotal {
```

Comment:
Nit:

```suggestion
		if (ac.Allocated+ac.Ready) == (bc.Allocated+bc.Ready) {
```
Seems like the intermediary variables aren't really necessary? WDYT?